### PR TITLE
DUX-528 sei: add base sources and prices

### PR DESCRIPTION
**Pull request by jeff-dude**
### Contribution type
Please check the type of contribution this pull request is for:

- [x] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

initiating foundation for adding sei spells.

### CODE REVIEW

File: models/evms/evms_info.sql [hunk 1]
```diff
@@ -50,4 +62,5 @@ FROM (VALUES
         , (60808, 'bob', 'BOB', 'Layer 2', 'Optimistic Rollup', 'ETH', 0x4200000000000000000000000000000000000006, 'https://explorer.gobob.xyz/', timestamp '2024-04-11 20:59:47', 'OP Stack', 'Ethereum Blobs', 'Ethereum')
         , (11155111, 'sepolia', 'Sepolia', 'Testnet', 'Testnet', 'tETH', 0x7b79995e5f793a07bc00c21412e50ecae098e7f9, 'https://sepolia.etherscan.io/', timestamp '2015-07-30 15:26:13', NULL, NULL, NULL)
         , (167000, 'taiko', 'Taiko', 'Layer 2', 'Optimistic Rollup', 'ETH', 0xa51894664a773981c6c112c43ce576f315d5b1b6, 'https://taikoscan.network/', timestamp '2024-03-29 07:39:12', 'OP Stack', 'Ethereum Blobs', 'Ethereum')
-        ) AS temp_table (chain_id, blockchain, name, chain_type, rollup_type, native_token_symbol, wrapped_native_token_address, explorer_link, first_block_time, codebase, data_availability, settlement)
+        , (1329, 'sei', 'Sei Network', 'Layer 1', NULL, 'SEI', 0xE30feDd158A2e3b13e9badaeABaFc5516e95e8C7, 'https://seitrace.com/?chain=pacific-1', timestamp '2024-05-27 13:14', NULL, NULL, NULL)
```
Comment (jeff-dude) [inline models/evms/evms_info.sql:65]: @hildobby doing my best impression of your work here. would you say this entry for SEI is accurate?

**Comment by jeff-dude** [comment]
@0xRobin @Hosuke 
plz help me finalize this with either an approval or recommendations for more 🙏 
the goal is to follow this pattern for new chains:
- PR similar to this to set the foundation
- PR to add dex(s) that read from some of this data
- PR to add nft(s) that read from some of this data

**Review by 0xRobin — APPROVED** [review]

**Comment by jeff-dude** [comment]
thanks both!
gonna get this merged in ✅

File: models/evms/evms_info.sql (hunk 1 above)
Comment (hildobby) [reply models/evms/evms_info.sql:65]: yeah that's perfect 👌

**Pull request merged**